From 390da4df31073b42b8161ebe117d36b0237c4c59 Mon Sep 17 00:00:00 2001
From: Aleksei Greshilov <aleksei.greshilov@cern.ch>
Date: Tue, 25 Oct 2022 18:10:54 -0400
Subject: [PATCH] i2c cdns and pm timeouts set to 4 ms

---
 drivers/i2c/busses/i2c-cadence.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/drivers/i2c/busses/i2c-cadence.c b/drivers/i2c/busses/i2c-cadence.c
index b51de7370810..0c0923f1f35b 100644
--- a/drivers/i2c/busses/i2c-cadence.c
+++ b/drivers/i2c/busses/i2c-cadence.c
@@ -113,9 +113,11 @@
 					 CDNS_I2C_IXR_DATA | \
 					 CDNS_I2C_IXR_COMP)
 
-#define CDNS_I2C_TIMEOUT		msecs_to_jiffies(1000)
+//#define CDNS_I2C_TIMEOUT		msecs_to_jiffies(1000)
+#define CDNS_I2C_TIMEOUT                msecs_to_jiffies(4)
 /* timeout for pm runtime autosuspend */
-#define CNDS_I2C_PM_TIMEOUT		1000	/* ms */
+//#define CNDS_I2C_PM_TIMEOUT		1000	/* ms */
+#define CNDS_I2C_PM_TIMEOUT             4       /* ms */
 
 #define CDNS_I2C_FIFO_DEPTH		16
 /* FIFO depth at which the DATA interrupt occurs */
